settlements/README.md

```diff
@@ -22,6 +22,17 @@ Please make sure your servers' clocks are correct, e.g. by using [NTP](https://e
 
 A Vipps merchant normally receives daily settlements. This can be changed to weekly, monthly settlements. See [the FAQ](https://www.vipps.no/sporsmal#bedriftspm) for more details.
 
+# Net and gross settlements
+
+Merchants with a "net settlement" contract receive the users' payments
+excluding the Vipps fees. On other words: The Vipps fees are deducted
+from the settlement amount.
+
+Merchants with a "gross settlement" contract receive the users' payments
+_including_ the Vipps fees, and are then invoiced for the Vipps fees.
```

Comment:
Merchants with a "gross settlement" contract receive the *full amount* of the users' payments

(drop the portion about Vipps fees)